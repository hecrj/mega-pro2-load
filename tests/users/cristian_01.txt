15

100
100
100
100
100
330
150
100
100
100
100
100
250
100
120

7

3 7 4 0 0 2 0 0 1 6 0 0 5 0 0

100
1 7
50
1 7
30
3 7 13 15
40
2 15 6
40
1 7
20
2 13 15
70
2 7 6


-1 7 50

-1 13 60

-1 15 61

-2 62

-3 63

-2 65

-3 66

-5 40 55

-5 40 70

-1 13 80

-5 40 90

-3 91

-1 15 100

-1 6 101

-1 13 102

-1 13 103

-3 104

-1 13 105

-1 13 107

-5 40 120

-5 100 101

-1 2 200

 -6
