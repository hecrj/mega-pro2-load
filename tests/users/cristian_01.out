Peticion procesada y servidores
0 1
2 3 7 
Peticion procesada y servidores
1 5
3 6 
Peticion procesada y servidores
2 3
4 
Peticiones pendientes
2 15 61 64
1 13 60 65
Servidores ocupados
3 1 2
4 2 1
6 1 2
Peticiones pendientes
0
Servidores ocupados
0
Pelicula mas solicitada
7 1
Pelicula mas solicitada
7 1
Peticion procesada y servidores
3 5
3 6 
Pelicula mas solicitada
13 2
Servidores ocupados
0
Peticion procesada y servidores
4 2
3 4 
Peticion procesada y servidores
5 5
7 
Peticion procesada y servidores
6 5
3 6 
Peticion procesada y servidores
7 0
Servidores ocupados
3 6 3
6 6 3
7 5 2
Peticion procesada y servidores
8 0
Peticion procesada y servidores
9 5
3 6 
Pelicula mas solicitada
13 4
Pelicula mas solicitada
6 1
Peticion procesada y servidores
10 0
